rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1132,6 +1132,40 @@ public Sha256Hash getBtcBlockchainBlockHashAtDepth(int depth) throws BlockStoreE
         return current.getHeader().getHash();
     }
 
+    public Long getBtcTransactionConfirmationsGetCost(Sha256Hash btcBlockHash) {
```

Comment:
Current implementation of `getBtcTransactionConfirmations` has a execution time proportional to the length of the array of transaction hashes, if a limit is not set then this method should consider that parameter.